src/components/favorites/Favorites.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { useContext } from "react";
+import VideoContext from "../../providers/VideoContext";
+import * as Style from "../styles/styleSheet";
+const Favorites = () => {
+  const { setFavoritesFlag } = useContext(VideoContext);
+  const { changinggSetChangingg } = useContext(VideoContext);
+  const { favoriteVideosMapUser } = useContext(VideoContext);
+  const { userSession } = useContext(VideoContext);
+  const { setFavoritesVideosArrayUser } = useContext(VideoContext);
+
+  const onFavorites = () => {
+    let favoriteVideos = favoriteVideosMapUser.get(userSession.user);
+
+    setFavoritesFlag(true);
+    changinggSetChangingg(false);
+    const favoriteVideosArray = [];
+
+    if (favoriteVideos) {
+      for (var clave of favoriteVideos.keys()) {
```

Comment:
Hello Ing. Diana,
I don't really understend your suggestion, because I am using a context Map in order to save globally the song information and access faster to the information when u need it, and in this part of the code as you suggest I am ussing a helper array 'favoriteVideosArray', just for showing the favorite videos in the ListVideos component (it receives an array and not a map), but if I am not understanding please let me know.
Thank you, regards.